src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -56,22 +56,38 @@ new Replay({
 
 ## Start and Stop Recording
 
-Replay recording only starts automatically when it's included in the integrations key when calling `Sentry.init`. Otherwise, you can initialize the plugin and manually call the `start()` method on the integration instance. To stop recording you can call the `stop()`:
+Replay recording starts automatically when it's included in the `integrations` key when calling `Sentry.init()` or when adding it via `addIntegration()`. Call `stop()` to stop recording and `start()` to resume recording:
```

Comment:
```suggestion
Replay recording starts automatically if it's included in the `integrations` key when calling `Sentry.init()` or added via `addIntegration()`. To stop recording, call `stop()`; to resume, call `start()`:
```

Reply:
super-l: In addition to Liza's suggestion, let's maybe replace key with array? 

```suggestion
Replay recording starts automatically if it's included in the `integrations` array when calling `Sentry.init()` or added via `addIntegration()`. To stop recording, call `stop()`; to resume, call `start()`: